SU-HSDO/stanford_mrc
# MRCD8-315 changed metatag and logo paths

# READY FOR REVIEW

# Summary
- Changed metatag and logo paths

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. Checkout this branch & same branch in [mrc_blt](https://github.com/SU-HSDO/mrc_blt/pull/14)
2. composer update
3. update database
4. ensure the logo and favicons still there.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
